build/gulp/plugins/util/checksumUtils.ts

```diff
@@ -0,0 +1,21 @@
+import * as fs from 'fs'
+import * as crypto from 'crypto'
+
+import config from '../../../../config'
+
+export const getInfoChecksum = (filename: string): string | null => {
+  try {
+    const buffer = fs.readFileSync(config.paths.docsSrc('componentInfo', filename))
+    const componentInfo = JSON.parse(buffer.toString())
+
+    return componentInfo.checksum
+  } catch (e) {}
```

Comment:
just to be sure - could you, please, suggest known cases this `catch` targets? Is it possible that some file other than component's info will be passed to this method in our workflow?

Reply:
`readFileSync` throws an error if file not exists, however it's a valid case for the first run

Reply:
got it, thanks!